Added translator.js

funkyClown opened the pull request:
Addaed the translator.js. We have two available languages (en/sl).
BUG: It dosen't translate all the word.

funkyClown committed:
Added translator.js

Addaed the translator.js. We have two available languages (en/sl).
BUG: It dosen't translate all the word.

the-tomazz commented:
Hei, dej še te prevode za javascript stringe (https://github.com/eEksperimenti/js-utils/tree/master/examples/translator#java_script_translate) - točka 5 ... pol pa dam merge :)

funkyClown committed:
All translations working

funkyClown committed:
Updated version of translator.js

funkyClown committed:
Updated translator.js, disabled button Averaging, and moved 3 buttons into collapse panel

translator.js:
-Updated Vocabulary
-Fixed a few bugs

index.html:
-disabled Averaging button
-added a new panel called Autoscale and moved Autoscale, Reset zoom and
AUTO into the panel.

funkyClown committed:
Revert "Updated translator.js, disabled button Averaging, and moved 3 buttons into collapse panel"

This reverts commit ec9f1afead886e437cbe3ffe2c4d22f807548ac7.

funkyClown committed:
Revert "Revert "Updated translator.js, disabled button Averaging, and moved 3 buttons into collapse panel""

This reverts commit 4c00a8662643cfb093b156df004ecdb54e33f761.

funkyClown committed:
...
